setup/MySQL/convert_mysql.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#Set database name
+if [ $# -eq 2 ]; then
+   MYDATABASE="atpdatabase"
+else
+   MYDATABASE=$3
+fi
+
+#Get MySQL username and password from the arguments
+USERNAME=$1
+PASSWORD=$2
+
+#Get PWD
+PWD=`pwd`
+
+echo "Creating $MYDATABASE"
```

Comment:
I'd like there to be similar status messages for all tables, i.e. "Importing players...", "Importing matches...", etc. Could you add these?